src/main/scala-2.11/markets/participants/strategies/CancellationStrategy.scala

```diff
@@ -5,7 +5,7 @@ import markets.orders.Order
 import scala.collection.mutable
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/2363092207/issues/source?bid=2921361&fileBranchId=3219434#l5)